prosemirror-jdita/src/document.ts

```diff
@@ -111,24 +136,110 @@ function defaultTravel(value: JDita, parent: JDita): any {
       type,
       attrs,
     };
+
     if (content) {
       result.content = content;
     }
   }
   return result;
 }
+
+/**
+ * `travel` is a recursive function that traverses
```

Comment:
`travel is a recursive function that traverses` -> `Traverses`

Reply:
Fixed with 25cfc27.